src/goto-instrument/externalize_symbols.h

```diff
@@ -0,0 +1,37 @@
+/// \file externalize_symbols.h
+/// \brief Remove static attribute from functions
+/// \author Kareem Khazem <karkhaz@karkhaz.com>
+
+#ifndef CPROVER_GOTO_INSTRUMENT_EXTERNALIZE_SYMBOLS_H
+#define CPROVER_GOTO_INSTRUMENT_EXTERNALIZE_SYMBOLS_H
+
+#include <goto-programs/goto_model.h>
+#include <util/message.h>
+
+#include <forward_list>
+#include <string>
+
+/// \brief Remove static attribute from functions
+class symbol_externalizert
+{
+public:
+  explicit symbol_externalizert(goto_modelt &goto_model, message_handlert &log)
+    : model(goto_model), log(log)
+  {
+  }
+
+  /// \brief Removes the static attribute from functions
+  ///
+  /// \return `true` on failure, `false` otherwise
+  bool externalize_symbols(const std::list<std::string> &);
+
+protected:
+  goto_modelt &model;
+  mutable messaget log;
+};
+
+#define OPT_STRIP_STATIC "(externalize-symbols):"
+#define HELP_STRIP_STATIC                                                      \
+  " --externalize-symbols fun[,fun...]  remove static from functions\n"
```

Comment:
Given the many meanings of `static` could this say something like "removes file-local flag from functions".